numba/unicode.py

```diff
@@ -1388,6 +1389,20 @@ def impl(s):
     return impl
 
 
+# https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Objects/unicodeobject.c#L11729-L11737    # noqa: E501
```

Comment:
I'm not sure how the source code referenced by this citation bears any relation to the implementation below?

Reply:
There is no relation, I thought it was necessary in any case to leave a link to the implementation in cpyton
Removed

Reply:
The links are algorithmic references and related to licensing:
https://github.com/numba/numba/blob/e4674eea90b451ae2141a20546727bef4ac8b352/LICENSES.third-party#L173-L179